frontend/src/component/onboarding/ConnectSdkDialog.tsx

```diff
@@ -77,13 +78,18 @@ export const ConnectSdkDialog = ({
     const [environment, setEnvironment] = useState<string | null>(null);
     const [apiKey, setApiKey] = useState<string | null>(null);
     const [stage, setStage] = useState<OnboardingStage>('select-sdk');
+    const { project } = useProjectOverview(projectId, {
```

Comment:
Isn't that a lot of data? Maybe we should have another endpoint for getting onboarding status?

Reply:
It is middle amount. I think it is fine for now, this is still beta. The thing is that this view is opened so rarely, so it makes barely no difference.